apps/pie-docs/src/foundations/iconography/overview.md

```diff
@@ -156,18 +156,46 @@ In this section you’ll find some factors you need to keep in mind when using i
 
 Abstract icons are more difficult to comprehend than literal icons. Instead of using metaphorical icons, try using those which convey the most basic idea or concept you’re trying to represent.
 
-{% contentPageImage {
-    src: "../../../assets/img/foundations/iconography/choosing-an-icon.svg",
-    mobileSrc: "../../../assets/img/foundations/iconography/choosing-an-icon-mobile.svg"
+{% usage {
+    do: {
+        type: usageTypes.image,
+        items: [{
+            src: "../../../assets/img/foundations/iconography/how-to-choose-an-icon-do.svg",
+            width: "234px",
+            alt: "illustration of how to choose an icon in a button"
```

Comment:
This is a great start - I think we might actually want the explain what the differences between the two images are as well (otherwise again we're relying on the user seeing the differences)